dynamo/vectorfield/clustering.py

```diff
@@ -196,18 +206,21 @@ def cluster_field(
                 adata,
                 adj_matrix_key="vf_feature_knn",
                 result_key="field_leiden",
+                **kwargs
```

Comment:
hdbscan doesn't have a resolution parameter

Reply:
I have a personal preference that it would be convenient to have a parameter named `resolution` controlling the clustering granularities. Although different clustering algorithms have different naming systems, hierarchical clustering may use names like depth, graph partition clustering may use resolution like louvain or leiden, k-means families used fixed number like k, they are all talking about the same thing - number of clusters.

So we can either 
1) use one "resolution" parameter, and tell users this is the controller of "number of clusters", and allow different underlying algorithm to have different behaviors to this parameters.
or 
2) add a bunches of parameters: depth, k, resolutions, likehood_cutoff, etc. , tell users to fill in proper parameters, and check the validity of all these parameters in each individual underlying algorithm.

I prefer a previous one. though algorithms like hdbscan has no `resolution` parameter, we can set one and translate it to the depth-like cutoffs in hdbscan calling wrappers. What do you think?